controller/handler_edge_ctrl/list_tunnel_services.go

```diff
@@ -83,7 +81,7 @@ func (self *listTunnelServicesHandler) listServices(ctx *listTunnelServicesReque
 		lastUpdate = self.appEnv.StartupTime
 	}
 
```

Comment:
This would skip if the timestamps were the same, so we were always sending service lists.